### ISSUE TITLE
Full scene description isn't read through by swiping on Mobile VO

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/friction#292]
**Test device**
iPad Air 2
**Operating System**
iPadOS 15.4.1
**Browser**
Safari
**Problem description**
For https://github.com/phetsims/qa/issues/791
If swiping right to read through the scene description with Mobile VO, a large part of the scene description is skipped. It can be accessed by swiping 2 fingers up to read from the beginning. 

**Visuals**

https://user-images.githubusercontent.com/41024075/161841746-ae708dc0-7d3d-4e13-b26d-09479a2e2bcb.MOV

**Comment by zepumph** [comment]
I see that the content that was skipped is all the dynamic content, which is nested in a div.

p -heard when reading line by line
div - not heard
p - heard

This seems like a voice over bug to me, we should test this as a p and see if it remains. Perhaps we will report to apple.

**Comment by zepumph** [comment]
@jessegreenberg noted that this has occurred before, in https://github.com/phetsims/john-travoltage/issues/449

**Comment by jessegreenberg** [comment]
I tried to get to the bottom of this in https://github.com/phetsims/a11y-research/issues/175 but did not succeed. I don't see anything in the PDOM that should be causing this. But I was not able to produce the bug outside of a PhET sim so it may be a problem with our infrastructure. But I feel like I have exhausted everything we can try.

I would recommend we add this to the list of known AT issues and not let this block publication.
